src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -717,6 +717,35 @@ g.test('depth_stencil_attachment')
     t.tryRenderPass(isValid, descriptor);
   });
 
+g.test('depth_stencil_attachment,depth_clear_value')
+  .desc(
+    `
+  Test that depthClearValue is invalid if the value is out of the range(0.0 and 1.0) when
+  depthLoadOp is 'clear'.
+  `
+  )
+  .paramsSimple([
+    { depthClearValue: -1.0 },
+    { depthClearValue: 0.0 },
+    { depthClearValue: 0.5 },
+    { depthClearValue: 1.0 },
+    { depthClearValue: 1.5 },
+  ])
```

Comment:
I realized while reviewing the implementation patch:
https://dawn-review.googlesource.com/c/dawn/+/96481
That this test needs to also test `.combine('depthLoadOp', ['load', 'clear']`. Because out-of-range values are only invalid if the `depthLoadOp` is `clear`:

> If this.depthLoadOp is "clear", this.depthClearValue must be between 0.0 and 1.0, inclusive.

@Gyuyoung can you make a PR to enhance this test? Thanks!

Reply:
We also need to test that it isn't validated when `depthLoadOp` is `undefined` (which is allowed only if there's no depth attachment or the attachment type doesn't have depth (e.g. it's `'stencil8'`).

Reply:
> I realized while reviewing the implementation patch: https://dawn-review.googlesource.com/c/dawn/+/96481 That this test needs to also test `.combine('depthLoadOp', ['load', 'clear']`. Because out-of-range values are only invalid if the `depthLoadOp` is `clear`:
> 
> > If this.depthLoadOp is "clear", this.depthClearValue must be between 0.0 and 1.0, inclusive.
> 
> @Gyuyoung can you make a PR to enhance this test? Thanks!

Ok, I upload a PR to apply `.combine('depthLoadOp', ['load', 'clear']` to https://github.com/gpuweb/cts/pull/1674. PTAL.

> We also need to test that it isn't validated when `depthLoadOp` is `undefined` (which is allowed only if there's no depth attachment or the attachment type doesn't have depth (e.g. it's `'stencil8'`).

It seems to me that `depth_stencil_attachment` test case can also check the case that `depthLoadOp` is `undefined`. 
Do you mean that we need to test it in `depth_stencil_attachment,depth_clear_value` again?

Please correct me if `depth_stencil_attachment` doesn't test the case that you want to test.